iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/ActivePowerControl.java

```diff
@@ -54,4 +56,25 @@ default String getName() {
      */
     void setParticipationFactor(double participationFactor);
 
+    /**
+     * @return if present, provides the overridden value of pmin to be used for active power control operations.
```

Comment:
Maybe you should be more precise here in the Javadoc to explain that it is used "only" during slack distribution if performed on generators ?

Reply:
pmin -> Pmin

Reply:
I used minP to be like the API.

Reply:
we agreed on minP with Anne, for the doc, after reviewing the doc.